openbb_platform/extensions/platform_api/openbb_platform_api/utils/widgets.py

```diff
@@ -91,9 +91,11 @@ def modify_query_schema(query_schema: list[dict], provider_value: str):
 
         modified_query_schema.append(_item)
 
-    modified_query_schema.append(
-        {"paramName": "provider", "value": provider_value, "show": False}
-    )
+    if provider_value != "custom":
```

Comment:
we might need the provider specified somewhere to fill in the metadata on the pro side. @andrewkenreich if we don't provide the provider in the params would this still work on the UI side?

Reply:
The metadata is under "source" for the Widget, which is still Custom. This just prevents creating a parameter that does not exist.

Reply:
![Screenshot 2024-09-18 at 9 37 35 AM](https://github.com/user-attachments/assets/be2a2a03-54fe-4090-af41-0669313a683d)